django/views/debug.py

```diff
@@ -294,7 +298,7 @@ def get_traceback_data(self):
             'settings': get_safe_settings(),
             'sys_executable': sys.executable,
             'sys_version_info': '%d.%d.%d' % sys.version_info[0:3],
-            'server_time': timezone.now(),
+            'server_time': localtime(datetime.utcnow().replace(tzinfo=utc)),
```

Comment:
Oh, right, now I understand.

It's unclear to me why the template engine would do anything with the datetime if `USE_TZ` is `False`, let alone crash; I believe that is the actual bug. (We probably had the same discussion about one year ago...)

Shall we deal with this at the DutH sprints?